### ISSUE TITLE
eval: extend embedding ablation with BGE-M3 / e5-large / KoSimCSE

### ISSUE THREAD

**Issue by hskim-solv** [hskim-solv/BidMate-DocAgent#174]
## Context
PR-05 (#156 / #148) landed the embedding ablation runner with a first measured comparison:
- MiniLM-L12-v2 vs intfloat/multilingual-e5-base
- `full` pipeline: 0pp delta on every primary metric (metadata-first robustness)
- `naive_baseline`: accuracy +18.8pp, groundedness +11.9pp

## Scope (this issue)
Extend the comparison with the larger candidates that did not fit in the original session:
- `BAAI/bge-m3` (~2GB, strong on Korean per MTEB)
- `intfloat/multilingual-e5-large` (~1.3GB)
- `BM-K/KoSimCSE-roberta-multitask` (~500MB)

## Procedure
Already-built runner accepts `--models`:

```bash
python3 scripts/run_embedding_ablation.py --models \
    sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2 \
    intfloat/multilingual-e5-base \
    intfloat/multilingual-e5-large \
    BAAI/bge-m3 \
    BM-K/KoSimCSE-roberta-multitask
```

## Environment requirements
- ~4GB free disk for model downloads
- `numpy<2` for PyTorch ABI compatibility (numpy 2.x triggers `_ARRAY_API not found`)
- Optional: GPU for faster inference (CPU works for n=42)

## Deliverables
- Append rows to the table in `docs/embedding-ablation.md`
- If `full` pipeline shows non-zero delta on any new model, propose a default change via ADR 0009
- If still 0pp on full but stronger naive_baseline lift, document the conclusion (metadata-first is robust to embedding choice across the modern multilingual family)

## Out of scope
- Korean-specific MTEB benchmark beyond this corpus
- Fine-tuning any embedding model on Korean RFP data

Follow-up to #148 / #156.
